packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -773,7 +856,12 @@ export function TransferPanelMain({
         />
       </div>
 
-      <NetworkContainer network={to}>
+      <NetworkContainer
+        network={to}
+        balanceFor={
+          walletAddress !== destinationAddress ? destinationAddress : undefined
```

Comment:
not entirely sure why
i put the connected wallet address into the destination address input field and the text "Showing balance for Destination Address: xxxxx" is showing up

it shouldn't be

Reply:
Fixed. Missed `toLowerCase`